prboom2/src/doomtype.h

```diff
@@ -116,6 +116,12 @@ typedef unsigned char byte;
   #define INLINE inline        /* use standard inline */
 #endif
 
+#ifdef WIN32
+#define PATHNAME_SEPARATOR '\\'
```

Comment:
Does this actually matter? I ask because there's a lot of code in this code base that builds paths with `/` and it seems to work fine 😄 

But maybe it doesn't work on every system, I don't know.

Reply:
DOS and Windows have always accepted `/` as a separator at the API level, but individual programs can choke on them due to using `/` to start command line switches, etc.  It's probably fine if it goes straight into a Win32 API call, but you could run into trouble if you pass it to a 3rd library which does something unexpected like pass it to an external program as a parameter.

Reply:
Seems like libzip does not make use of external programs, so I'll just revert that change. Thanks for the insight 👍